src/fontcache/freetypefontcache.cpp

```diff
@@ -303,7 +303,8 @@ const Sprite *FreeTypeFontCache::InternalGetGlyph(GlyphID key, bool aa)
 	}
 
 	GlyphEntry new_glyph;
-	new_glyph.sprite = BlitterFactory::GetCurrentBlitter()->Encode(spritecollection, SimpleSpriteAlloc);
+	SimpleSpriteAllocator allocator;
+	new_glyph.sprite = BlitterFactory::GetCurrentBlitter()->Encode(spritecollection, allocator);
```

Comment:
Depending on the colour of your bikeshed, you could also make this (and the other lines) `new_glyph.sprite = BlitterFactory::GetCurrentBlitter()->Encode(SimpleSpriteAllocator{}, allocator);`

Reply:
The `allocator` parameter would have to be `const` to allow this, and then the `Allocate()` method would also have to be `const`, which I'm not sure is desirable.

Reply:
Hmm, yes. Or I think you could make it a `&&` :)

Reply:
Tricky given it's passed between functions, sometimes from a pointer (optional references aren't a thing afaik.)